build.gradle.kts

```
@@ -86,9 +86,9 @@ java {
 
 tasks.withType<Jar>().configureEach {
     manifest.attributes(
-        "Implementation-Title" to project.name,
-        "Implementation-Vendor" to metadata.organization.get().name.get(),
-        "Implementation-Version" to project.version
+            "Implementation-Title" to project.name,
+            "Implementation-Vendor" to metadata.organization.get().name.get(),
+            "Implementation-Version" to project.version
```

```suggestion
        "Implementation-Title" to project.name,
        "Implementation-Vendor" to metadata.organization.get().name.get(),
        "Implementation-Version" to project.version
```
Sometimes IntelliJ messes up the code style and seems to apply the Java code style to Kotlin scripts.

---

I think you need to remove some files in the .idea folder and then restart IntelliJ iirc.

---

FIxed.